### ISSUE TITLE
monitoring_enable_managed_prometheus field not recognised on v"24.1.0"

### ISSUE THREAD

**Issue by s04** [terraform-google-modules/terraform-google-kubernetes-engine#1540]
### TL;DR

Trying to set the "monitoring_enable_managed_prometheus" equal to true on version 24.1.0 leads to "An argument named "monitoring_enable_managed_prometheus" is not expected here." from terraform.

### Expected behavior

I expect it work according to the [README.md](https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/blob/master/README.md) which allows this field.

Weirdly, this parameter is not listed on the terraform module readme on the terraform website. Shouldn't they be the same?

https://registry.terraform.io/modules/terraform-google-modules/kubernetes-engine/google/24.1.0

See here ^ no "monitoring_enable_managed_prometheus" field.

### Observed behavior

 "An argument named "monitoring_enable_managed_prometheus" is not expected here." from terraform.

### Terraform Configuration

```hcl
module "gke" {
  source                 = "terraform-google-modules/kubernetes-engine/google"
  version                = "24.1.0"

  project_id                        = var.project_id
  name                              = var.cluster_name
  regional                          = true
  region                            = var.region
  network                           = module.gke-network.network_name
  subnetwork                        = module.gke-network.subnets_names[0]
  ip_range_pods                     = module.gke-network.subnets_secondary_ranges[0].*.range_name[0]
  ip_range_services                 = module.gke-network.subnets_secondary_ranges[0].*.range_name[1]
  create_service_account            = true
  horizontal_pod_autoscaling        = true
  remove_default_node_pool          = true
  disable_legacy_metadata_endpoints = true
  grant_registry_access	            = true
  cluster_autoscaling               = var.cluster_autoscaling
  monitoring_service = "monitoring.googleapis.com/kubernetes"
  monitoring_enable_managed_prometheus = true
}
```

### Terraform Version

```sh
Terraform v1.3.7
on darwin_arm64
+ provider registry.terraform.io/hashicorp/google v4.49.0
+ provider registry.terraform.io/hashicorp/google-beta v4.49.0
+ provider registry.terraform.io/hashicorp/kubernetes v2.16.1
+ provider registry.terraform.io/hashicorp/random v3.4.3
```

**Comment by IIBenII** [comment]
`monitoring_enable_managed_prometheus` is enable only for beta module in version "24.1.0". It will be available in GA with the release "25.0.0": https://github.com/terraform-google-modules/terraform-google-kubernetes-engine/pull/1512
